src/test/java/Homework16.java

```diff
@@ -0,0 +1,26 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+    @Test
+    public static void Navigation() {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+
+        WebElement registrationlink = driver.findElement(By.cssSelector("[id='hel']"));
+        registrationLink.click();
+
+        String registrationUrl = "http://bbb.testpro.io/registration.php";
+        Assert.assertEquals(driver.getCurrentUrl(), registrationUrl);
```

Comment:
Test failed! You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221778933-4723cc69-e8bb-4aa8-93a5-93fa3b0ebcab.png)

![image](https://user-images.githubusercontent.com/104761482/221779044-35a33030-9b45-495d-b0af-3a3554112912.png)